src/components/TokenBalance.tsx

```diff
@@ -119,14 +131,62 @@ function useErrorMessageWithRefresh() {
   }, [shouldShowError])
 }
 
-export function AssetsTokenBalance() {
+export function AssetsTokenBalance({ showInfo }: { showInfo: boolean }) {
   const { t } = useTranslation()
 
+  const [infoVisible, setInfoVisible] = useState(false)
+  const [shouldRenderInfoComponent, setShouldRenderInfoComponent] = useState(false)
+  const infoOpacity = useSharedValue(0)
+
+  useShowOrHideAnimation(
+    infoOpacity,
+    infoVisible,
+    () => {
+      setShouldRenderInfoComponent(true)
+    },
+    () => {
+      setShouldRenderInfoComponent(false)
+    }
+  )
+
+  const animatedStyles = useAnimatedStyle(() => {
+    return {
+      opacity: infoOpacity.value,
+    }
+  })
+
+  const toggleInfoVisible = () => {
+    setInfoVisible((prev) => !prev)
+  }
+
+  const handleDismissInfo = () => {
+    setInfoVisible(false)
+  }
+
   return (
-    <View testID="AssetsTokenBalance">
-      <Text style={styles.totalAssets}>{t('totalAssets')}</Text>
-      <TokenBalance singleTokenViewEnabled={false} />
-    </View>
+    <TouchableWithoutFeedback onPress={handleDismissInfo}>
+      <View testID="AssetsTokenBalance">
+        <View style={styles.row}>
+          <Text style={styles.totalAssets}>{t('totalAssets')}</Text>
+          {showInfo && (
+            <TouchableOpacity
```

Comment:
Ideally this would use `src/components/Touchable` to respect the platform convention (opacity on iOS and Ripple on Android). But we haven't been very consistent so far through the app.

Reply:
for this component i opted for the same behaviour on the home screen - i think TouchableOpacity works better here because the info icon is round. i did try with the Touchable component but the shadow is square and grey on android...